_includes/checkpoints/checkpoint2.py

```diff
@@ -1,19 +1,38 @@
-# need to import the "math" module to use different math functions
+"""
+Checkpoint 2: Modularize a repeated problem
+-------------------------------------------
+
+Create a function to calculate the result for any initial flux, shield
+thickness, and attenuation coefficient.
+"""
+
 import math as m
 
+
 def calc_phi(phi_0, mu, x):
+    """Calculates final flux after shield of thickness x with
+    attenuation coefficient mu given the starting flux phi_0.
 
-    phi = phi_0 * m.exp(-mu*x)
+    Input:
+    ------
+        phi_0: float, initial flux value (photons/cm**2-s)
+        mu: float, attenuation coefficient (1/cm)
+        x: float, shield thickness (cm)
 
-    return(phi)
+    Returns:
+    --------
+        phi: float, final flux value (photons/cm**2-s)
+    """
 
+    phi = phi_0 * m.exp(-mu*x)
+    return(phi)
 
 
-# Lead shield 5 cm thick
-phi_lead = calc_phi(1.e10, 0.125, 5.0)
+# Lead shield
+phi_lead = calc_phi(1.e10, 0.5182, 5.0)
```

Comment:
Keeping these parameters passed into the function as variables would be a smaller diff compared to the previous checkpoint - less typing for the learners

Reply:
Just updated that and made them all variables again.